### ISSUE TITLE
A read error happens when trying to view certain cjxl.exe converted JPEG XL images

### ISSUE THREAD

**Issue by Funkmiser** [dlemstra/Magick.NET#1196]
### Magick.NET version

11.1.2

### Environment (Operating system, version and so on)

Windows 11 x64 Version 21H2 (OS Build 22000.675)

### Description

When using the cjxl.exe encoder to convert a jpg to jxl and check the resulting file sizes I noticed that some of the resulting images would throw an error through the Magick.NET-Q16-OpenMP dll. I'm using the latest version of [ImageGlass Kobe](https://imageglass.org/) (8.6.6.6) as the image viewer and when the error occurs the error text pops up instead of the image, I've attached a picture below of the error message:
![Screenshot 2022-06-04 175712](https://user-images.githubusercontent.com/53496265/172028154-ed5eb245-99c5-4e6f-b389-0985206abf76.png)
As you can see it complains about being unable to read the image data.

This seems to only happen to certain jxl conversions, with the test image here: [test image and encodings.zip](https://github.com/dlemstra/Magick.NET/files/8839079/test.image.and.encodings.zip) you can see that when using the cjxl.exe distance=0 option then the error gets thrown but when using the distance=1 and above option no error is thrown. It's using the default effort=7 value. I've used djxl.exe to decode the distance=0 conversion back to a jpg so the included decoder seems to read it fine.
And there are other png and jpg images I've converted (that had smaller file sizes than the test image) using the distance=0 option and those resulting images can be read fine.

I don't know exactly what version of Magick.NET that ImageGlass Kobe is using but I downloaded the NuGet package for Magick.NET and replaced the dll that ImageGlass was using with the 11.1.2 version one as that seemed to be the latest version and the error still pops up.

I got the (jxl-x64-windows-static) cjxl.exe [June 01, 2022] from https://artifacts.lucaversari.it/libjxl/libjxl/latest/ as this appears to be a backup for the build artifacts by one of the libjxl developers but it also happens with the v0.6.1 version from https://github.com/libjxl/libjxl/releases

#1049 seemed to have a similar issue but the libjxl issue that was linked from there looks to have been resolved in November 2021.

### Steps to Reproduce

1. Take the 'PC Engine Fan ...' test jpg image from the linked zip above and use the cjxl.exe (v0.6.1 or otherwise) to convert to a jxl image using the -d 0 (distance=0) option.
```
.\cjxl.exe '.\PC Engine Fan - 1989-12 December - Genpei Toma Den Vinyl Soundtrack - Cover Front.jpg' 'output d0.jxl' -d 0
```
2. Get an error thrown when trying to read the resulting image.

**Comment by dlemstra** [comment]
This looks like a bug inside @ImageMagick and I just pushed a patch for this to resolve this issue in that repository. I will try to publish a new release of MagickNET next week.

**Comment by Funkmiser** [comment]
ImageGlass did a hotfix release [v8.6.7.13](https://github.com/d2phap/ImageGlass/releases/tag/8.6.7.13) that includes the just released v11.2.0 Magick.Net libraries and that looks to have fixed the decoding of the previously not working jpeg xl image. 👍
